packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -361,6 +361,13 @@ class _RenderLayoutBuilder extends RenderBox with RenderObjectWithChildMixin<Ren
     }
   }
 
+  @override
+  double computeDistanceToActualBaseline(TextBaseline baseline) {
+    if (child != null)
+      child.getDistanceToActualBaseline(baseline);
```

Comment:
`return child.getDistanceToActualBaseline(baseline);` ?

Reply:
I was afraid that changing how baseline is calculated may have adverse effects. That's why I only called `getDiatanceToActualBasline` to pass the baseline to child.

Should I override how the baseline is computed by returning it?

Reply:
I decided to only pass the baseline to child because of this [issue I faced](https://github.com/flutter/flutter/pull/63887#issuecomment-675278772)

Reply:
I did test it out, and it has the adverse effect on how child is laid out on y coordinate. Wouldn't that make it a breaking change?

Reply:
let me take a step back, May I know what is the use case of this change? It seems weird that we call getDiatanceToActualBasline of the child if we don't want to use the result. That will be a weird side affect.

Reply:
If you check out [the comment](https://github.com/flutter/flutter/pull/63887#issuecomment-675278772) which describes the blockage for the PR. Hexie suggested providing Ink widget a private RenderObject. So instead of providing a private RenderObject, I suggested that what if we implement the baseline logic for LayoutBuilder instead of rewriting the most of the code for Ink widget.

Reply:
Ah So this is for a failing test. I think we should return the child, otherwise according to the test it will no longer baseline the list title correctly.

Reply:
But I do agree with you that there is no point if we are not returning the child's baseline since then Baseline won't be able to properly define the y coordinate for a child. I'm just concern that doing so may be a breaking change.

Reply:
I think this is more programmatically correct, so i think it is ok if this is a breaking change.

Reply:
So, is the PR good to go without a breaking change design doc or we should handle it with a flag property? I can write a design doc for it, if needed.

Reply:
We only consider it a breaking change when it break contributed tests https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

So you should make the change first to see if it does break tests

Reply:
I have already made the changes to return the child's baseline, and it's not breaking any test.